carbonmark/components/pages/Retire/RetirementStatusModal/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { ButtonPrimary, Text } from "@klimadao/lib/components";
+import { Trans } from "@lingui/macro";
+import { CelebrationOutlined } from "@mui/icons-material";
+import { Modal } from "components/Modal";
+import Link from "next/link";
+import { FC } from "react";
+import * as styles from "./styles";
+
+type Props = {
+  retirementUrl: string;
+};
+
+export const RetirementStatusModal: FC<Props> = (props) => (
+  <Modal
+    title={
+      <div className={styles.title}>
+        <CelebrationOutlined fontSize="large" />
+
+        <Trans id="offset.successModal.title">
+          <Text t="h4">Retirement Successful!</Text>
+        </Trans>
+      </div>
+    }
+  >
+    <div className={styles.modalContent}>
+      <Text t="body6">
+        <Trans id="offset.successModal.body1">
+          Thank you for supporting the planet! View transaction on{" "}
+          <Link href={props.retirementUrl}>polygonscan.</Link>
+        </Trans>
+      </Text>
+
+      <ButtonPrimary
+        href={props.retirementUrl}
+        className={styles.viewButton}
+        target="_blank"
+        label={
+          <Trans id="offset.successModal.cta_1">
+            VIEW AND SHARE CERTIFICATE
```

Comment:
As best we can lets keep strings In a consistent case, and then transform using CSS `text-transform`

```suggestion
            View and share certificate
```